build/_build/Build.Steps.cs

```diff
@@ -75,6 +75,7 @@ partial class Build
     Project[] ParallelIntegrationTests => new[]
     {
         Solution.GetProject(Projects.TraceIntegrationTests),
+        Solution.GetProject(Projects.AppSecIntegrationTests),
```

Comment:
Is this definitely safe to run in parallel? It uses IIS, and touches various samples, so just wondered. It shouldn't be a problem as the ClrProfiler tests aren't running in parallel with it, just wanted to double check :)

Reply:
It does use IIS, I wasn't sure of the criteria for running in parallel, so I just guessed that it would be okay.

I guess will just see if it works for now. I think we'll probably move it to it's own build step long term, but we have too few tests to justify that at the moment.

Reply:
Yeah, that makes sense. There isn't hard and fast criteria for running in parallel AFAIK (the approach here is what existed pre-Nukeification), but if it's using IIS it definitely _can't_ run at the same time as the ClrProfiler tests.